src/Evolution/Initialization/Evolution.hpp

```diff
@@ -67,60 +67,37 @@ namespace Actions {
 ///   * Tags::TimeStepId
 ///   * `Tags::Next<Tags::TimeStepId>`
 ///   * Tags::TimeStep
-///   * `db::add_tag_prefix<Tags::dt, variables_tag>`
-///   * `Tags::HistoryEvolvedVariables<variables_tag, dt_variables_tag>`
 ///   * Tags::Time
-///   * Tags::ComputeDeriv  (for non-conservative systems)
-///   * Tags::ComputeDiv (for conservative systems)
 /// - Removes: nothing
 /// - Modifies: nothing
 ///
 /// \note HistoryEvolvedVariables is allocated, but needs to be initialized
 template <typename Metavariables>
-struct Evolution {
+struct TimeAndTimeStep {
   using initialization_tags =
       tmpl::list<Tags::InitialTime, Tags::InitialTimeDelta,
                  Tags::InitialSlabSize<Metavariables::local_time_stepping>>;
 
-  static constexpr size_t dim = Metavariables::volume_dim;
-  using variables_tag = typename Metavariables::system::variables_tag;
-  using dt_variables_tag = db::add_tag_prefix<::Tags::dt, variables_tag>;
+  template <
+      typename DbTagsList, typename... InboxTags, typename ArrayIndex,
+      typename ActionList, typename ParallelComponent,
+      Requires<tmpl::list_contains_v<
+                   typename db::DataBox<DbTagsList>::simple_item_tags,
+                   Initialization::Tags::InitialTime> and
 
```

Comment:
Yes, I found it a lot easier to read, but am fine changing if people disagree :)

Reply:
This could be written `cpp17::is_same_v<tmpl::list_difference<initialization_tags, typename db::DataBox<DbTagsList>::simple_item_tags>, tmpl::list<>>` or similar, which is shorter but a bit harder to understand.  I'll leave that up to you.  (Maybe we should add a `tmpl::is_subset` or `tmpl::contains_all` or something.)